sympy/discrete/transforms.py

```diff
@@ -85,9 +94,53 @@ def fft(seq, dps=None):
 
     seq : iterable
         The sequence on which **DFT** is to be applied.
+
     dps : Integer
         Specifies the number of decimal digits for precision.
 
+    fourier_parameters : (Integer, Integer)
+        Specifies the formula used for fourier transformation.
+
+        (1, 1) is the SymPy's defaults.
+
+        (1, -1) is equivalent to the NumPy and MATLAB's default settings for
+        the ``fft`` and ``ifft``.
+
+        (0, 1) is equivalent to the Mathematica's default settings for
+        ``Fourier`` and ``InverseFourier``.
+
+        (0, -1) is equivalent to the Maple's default settings for
+        ``FourierTransform`` and ``InverseFourierTransform``
+
+        For further information about the options and formulas, see ``Notes``
+        section.
+
+    Notes
+    =====
+
+    The formula for computing fourier transformation may vary across each
+    libraries, and caution should be taken.
+
+    SymPy's default implementation uses the formula
+
+    `A_{k} = \sum_{n=0}^{N-1} a_{n} e^{2 \pi i*n*k/N}` for ``fft``
```

Comment:
Perhaps, this could be written in a more general form, in terms of `\omega` (with its value indicated separately).